src/liboslexec/opfmt.cpp

```diff
@@ -31,10 +31,12 @@ namespace pvt {
 // Shims to convert llvm gen to rs free function C++ parameter types
 // and forward on calls to re free functions.
 
+// TODO this can be removed now that batched supports ustringhash_pod
```

Comment:
> I don't think this impl should be changed at all. Just people calling it maybe don't need to call it. Then if there are no callers left, it can be removed.

Yes. I changed it because I didn't want atm. to go through all the code calling it, and instead made the change here in a single place.